src/Wb/BigRegister/SoapClient/ResponseParser.php

```diff
@@ -144,65 +145,61 @@ protected function getProfession($code)
-            43 => 'maatschappij en gezondheid',
-            44 => 'sportgeneeskunde',
-            45 => 'urologie',
-            46 => 'obstetrie en gynaecologie',
-            47 => 'verpleeghuisgeneeskunde',
-            48 => 'arbeid en gezondheid - verzekeringsgeneeskunde',
-            50 => 'zenuw- en zielsziekten',
-            53 => 'dento-maxillaire orthopaedie',
-            54 => 'mondziekten en kaakchirurgie',
-            55 => 'maatschappij en gezondheid',
-            56 => 'medische zorg voor verstandelijke gehandicapten',
-            60 => 'ziekenhuisfarmacie',
-            61 => 'klinische psychologie',
-            62 => 'interne geneeskunde-allergologie',
-            63 => 'klinische neuropsychologie',
-            65 => 'verpl. spec. prev. zorg bij som. aandoeningen',
-            66 => 'verpl. spec. acute zorg bij som. aandoeningen',
-            67 => 'verpl. spec. intensieve zorg bij som. aandoeningen',
-            68 => 'verpl. spec. chronische zorg bij som. aandoeningen',
-            69 => 'verpl. spec. geestelijke gezondheidszorg',
-            70 => 'jeugdgezondheidszorg',
-            71 => 'spoedeisendehulp',
-            74 => 'sportgeneeskunde',
-            75 => 'openbaar apotheker'
+            2 => 'Allergologie (allergoloog)',
+            3 => 'Anesthesiologie (anesthesioloog)',
+            4 => 'Huisartsgeneeskunde met apotheek (Apoth. Huisarts)',
+            8 => 'Arbeid en gezond - bedrijfsgeneeskunde',
+            10 => 'Cardiologie (cardioloog)',
+            11 => 'Cardio-thoracale chirurgie',
+            12 => 'Dermatologie en venerologie (dermatoloog)',
+            13 => 'Maag-darm-leverziekten (maag-darm-leverarts)',
+            14 => 'Heelkunde (chirurg)',
+            15 => 'Huisartsgeneeskunde (huisarts)',
+            16 => 'Interne geneeskunde (internist)',
+            18 => 'Keel-, neus- en oorheelkunde (kno-arts)',
+            19 => 'Kindergeneeskunde (kinderarts)',
+            20 => 'Klinische chemie (arts klinische chemie)',
+            21 => 'Klinische genetica (klinisch geneticus)',
+            22 => 'Klinische geriatrie (klinisch geriater)',
+            23 => 'Longziekten en tuberculose (longarts)',
+            24 => 'Medische microbiologie (arts-microbioloog)',
+            25 => 'Neurochirurgie (neurochirurg)',
+            26 => 'Neurologie (neuroloog)',
+            30 => 'Nucleaire geneeskunde (nucleair geneeskundige)',
+            31 => 'Oogheelkunde (oogarts)',
+            32 => 'Orthopedie (orthopeed)',
+            33 => 'Pathologie (patholoog)',
+            34 => 'Plastische chirurgie (plastisch chirurg)',
+            35 => 'Psychiatrie (psychiater)',
+            39 => 'Radiologie (radioloog)',
+            40 => 'Radiotherapie (radiotherapeut)',
+            41 => 'Reumatologie (reumatoloog)',
+            42 => 'Revalidatiegeneeskunde (revalidatiearts)',
+            43 => 'Maatschappij en gezondheid  (beëindigd per 01-01-2007)',
```

Comment:
Can you update this to `Maatschappij en gezondheid  (beëindigd per 01-01-2007)` with one space